.github/workflows/ci-comment.yml

```diff
@@ -12,7 +13,7 @@ on:
       root-sbd:
         type: string
         required: false
-        default: ${{ inputs.model }}
+        # default: ${{ inputs.model }}
```

Comment:
Should this just be deleted? Or a comment added saying this isn't possible, so workflow below effectively does this?

Reply:
Good point. Will update.

Reply:
Done in https://github.com/ACCESS-NRI/build-cd/pull/192/commits/d6c41a87cc69ea1f63e5b7303a714d4602aa85e0